evaluation_script/main.py

```diff
@@ -153,7 +141,18 @@ def populate_ground_truth(infile):
     Note: This skips the first line of a ground truth file (header).
     """
     index = populate_index_map(infile)
```

Comment:
Might be worth adding a blank line somewhere here. My reason is that when I look at this block of code, the `global_metrics` and `predicted_keys` declarations looks like they are in a docstring:
```
    """
    global_metrics = Metrics()
    predicted_keys = set()
    """ predicted_keys serves dual purpose.
```

They aren't of course. I didn't find any python style guide that suggested what to do, but perhaps a blank line would help. The other possibility would be to use regular comments to document local variables (which these now are). It may be that this is a more common way to write python, not sure.

Reply:
Addressed in latest update.